zmon-controller-app/src/main/java/org/zalando/zmon/api/EntityApi.java

```diff
@@ -57,12 +57,19 @@ public EntityApi(EntitySProcService entitySprocs, ObjectMapper mapper, DefaultZM
     @ResponseStatus(HttpStatus.OK)
     @ResponseBody
     @RequestMapping(value = {"/", ""}, method = {RequestMethod.PUT, RequestMethod.POST})
-    public void addEntity(@RequestBody JsonNode entity) {
-        if (entity.has("type") && "zmon_config".equals(entity.get("type").textValue())) {
-            if (!authService.hasAdminAuthority()) {
-                throw new AccessDeniedException("No ADMIN privileges present to update configuration.");
+    public ResponseEntity<String> addEntity(@RequestBody JsonNode entity) {
+
+        if (entity.has("type")){
+            if ("global".equals(entity.get("type").textValue().toLowerCase())){
+                return new ResponseEntity<>("Creating entity with type - GLOBAL is not allowed.", HttpStatus.BAD_REQUEST);
```

Comment:
Maybe this should HttpStatus.FORBIDDEN instead...

Reply:
+1